server/sonar-db-dao/src/main/protobuf/db-project_branches.proto

```diff
@@ -32,7 +32,7 @@ option optimize_for = SPEED;
 message PullRequestData {
   required string branch = 1;
```

Comment:
I strongly suggest to use only optional and not use required. Recommendation from Protobuf spec. In Protobuf 3 it has been removed to only have optional elements

Reply:
I didn't know, thanks. I changed to `syntax = "proto3"` now.